### ISSUE TITLE
[ACS - Bracelet] Flower Bracelet

### ISSUE THREAD

**Issue by Velius94** [Monika-After-Story/MonikaModDev#4381]
Because smol things are precious.
Comes in two flavors, light and dark.

![bracelet light](https://user-images.githubusercontent.com/1176163/60610716-ba85ea00-9dc4-11e9-891f-299d5ab18cae.png)
![bracelet dark](https://user-images.githubusercontent.com/1176163/60610715-ba85ea00-9dc4-11e9-858f-79f426a1557b.png)

PSDs:
SOON

**Comment by ThePotatoGuy** [comment]
slated for 0.10

**Comment by ThePotatoGuy** [comment]
*0.10.2

**Comment by ThePotatoGuy** [comment]
@Velius94  can you repload psds?

**Comment by Velius94** [comment]
up again

**Comment by Orcaramelo** [comment]
Hey wait are the psds already refitted?

**Comment by Velius94** [comment]
Nah, these are originals. Might want to upload your modifications.

**Comment by ThePotatoGuy** [comment]
on hold because of #5027
